modules/core/src/main/java/org/apache/fluo/core/impl/FluoConfigurationImpl.java

```diff
@@ -143,6 +143,45 @@ public static long getTxIfoCacheTimeout(FluoConfiguration conf, TimeUnit tu) {
     }
     return tu.convert(millis, TimeUnit.MILLISECONDS);
   }
+  public static final String VISIBILITY_CACHE_SIZE = FLUO_IMPL_PREFIX + ".visibility.cache.size.mb";
+  public static final long VISIBILITY_CACHE_SIZE_DEFAULT = 10000000;
```

Comment:
I think in java 7 or 8 the ability to add underscores in numbers was added.  So could change this to `10_000_000` to make it a bit more readable.